pkg/dockerregistry/server/middlewares.go

```diff
@@ -0,0 +1,88 @@
+package server
```

Comment:
Can we name it `register.go` or `middlewareinit.go` or just `init.go`?

Can we also rename `repositorymiddleware` to `repository`?